src/App.tsx

```diff
@@ -1,5 +1,49 @@
+import colors from './colors.json';
+
+function ColorObject({ colorInfo }) {
+    return (
+        <div
+            style={{
+                color: `#${colorInfo.hex}`,
+            }}
+        >
+            {colorInfo.name}
+        </div>
+    );
+}
+
+function ColorObjectWithComp({ colorInfo }) {
```

Comment:
Great job breaking out components for these different visual components. It makes the code more more readable and organized.